packages/core-mobile/e2e/configs/regressionConfig.json

```diff
@@ -25,6 +25,7 @@
   },
   "testPathIgnorePatterns": [
     "<rootDir>/tests/plusIcon/send/sendBitcoin.e2e.ts",
-    "<rootDir>/tests/notification/sendNotification.e2e.ts"
+    "<rootDir>/tests/notification/sendNotification.e2e.ts",
+    "<rootDir>/tests/settings/notification.e2e.smoke.ts"
```

Comment:
We can't do notification testing on the external build, which disabled the notification feature.